tasks/ably/sse.go

```diff
@@ -0,0 +1,57 @@
+package ably
+
+import (
+	"context"
+	"fmt"
+	"net/url"
+
+	"github.com/ably/ably-boomer/tasks"
+	"github.com/ably/ably-go/ably/proto"
+	"github.com/r3labs/sse"
+)
+
+type wrappedSSEClient struct {
+	*sse.Client
+}
+
+func (wc *wrappedSSEClient) Subscribe(ctx context.Context, msgHandler func(message *proto.Message)) error {
```

Comment:
Missing comment on exported func (here and other places)

Reply:
it's not an exported function. It implements a public interface

Reply:
Ah yes, indeed.